src/test-utils.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+import { IntlProvider } from 'react-intl';
+import messages from '../i18n/core.json';
+
+const customRender = (node, ...options) =>
+  render(
+    <IntlProvider locale="en" messages={messages}>
+      {node}
+    </IntlProvider>,
+    ...options
+  );
+
+// re-export everything
+export * from 'react-testing-library';
```

Comment:
All other exports are reexported so that we can do

```
import { render, fireEvent } from '../test-utils'
```

in tests. This means that we don't even need to import from `react-testing-library` in tests anymore. This is the recommended way of `react-testing-library`.